pwd_guess.py

```diff
@@ -965,31 +990,62 @@ def training_step(self, x_all, y_all, w_all):
             x_val, y_val, sample_weight=w_val)
         return (train_loss, train_accuracy, test_loss, test_accuracy)
 
-    def train_model_generation(self):
+    def early_stopping(self,smooth_loss, serializer):
```

Comment:
Please use the pylint style here. To run it on this file, you can do "pylint --rcfile=pylintrc pwd_guess.py" There are a handful of other style comments that the pylint picks up too.